css/webstyle.css

```diff
@@ -0,0 +1,346 @@
+body {
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  height: 7.86vh;
+  background-color: #fff;
+  align-items: baseline;
+}
+
+h3 {
+  flex: 1;
+  margin-left: 10%;
+  color: #6070ff;
+}
+
+nav {
+  flex: 2;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 10%;
+}
+
+.hide {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  padding-inline-start: 0;
+  align-self: stretch;
+}
+
+ul :first-child {
+  padding-inline-start: 0;
+}
+
+li {
+  list-style: none;
+  padding-left: 12px;
+  padding-right: 12px;
+  margin-right: 8px;
+  font-weight: 500;
+}
+
+#hamb {
+  display: none;
+}
+
+main {
+  background-color: #fff;
+  background-image: url(../images/Headerbg.png);
+  background-size: 96%;
+  background-repeat: no-repeat;
+  background-position: 0 31%;
+  border-bottom-left-radius: 8.5% 8.5%;
```

Comment:
- [ ] I don't see where your social icons CSS attribute, they look too small and missing the **angelelist & meduim.**

Reply:
not a point to have them all i could add how much I want. And that will not make any changes.  Since that is mine portfolio and Im avalable only on those three thats why I didn't included those